bower.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-copy-to-clipboard",
+  "version": "1.0.1",
+  "homepage": "https://github.com/nkbt/react-copy-to-clipboard",
+  "authors": [
+    "Nik Butenko <nik@butenko.me> (http://butenko.me/)"
+  ],
+  "description": "Copy-to-clipboard React component",
+  "main": "lib/index.js",
```

Comment:
I wish it is easy as adding config file. The problem with bower is that we also need to check in build artifact into a repo.

As you can see you use `lib/index.js` here which is not in the repo. It is compiled and pushed to the npm package on publish because npm stores build artifacts for you. Not the case of bower.

There is some discussion about bower https://github.com/chenglou/react-motion/issues/59 so you can get a bit more context.

And more discussion in Slack.

So what we also need to do is to add one more webpack build process and push build artifact into a repo.

Do you feel like doing it? Alternatively I can merge this PR in and do rest by myself.